Prog/AddNew/TOPA_match_notices_property.sas

```diff
@@ -0,0 +1,101 @@
+/**************************************************************************
+ Program:  TOPA_match_notices_property.sas
+ Library:  PresCat
+ Project:  Urban-Greater DC
+ Author:   Elizabeth Burton
+ Created:  12/13/2022
+ Version:  SAS 9.4
+ Environment:  Local Windows session (desktop)
+ 
+ Description:  Match TOPA notices to the same properties
+ 
+ Modifications:
+**************************************************************************/
+
+%include "\\sas1\DCdata\SAS\Inc\StdLocal.sas";
+
+** Define libraries **;
+%DCData_lib( PresCat )
+%DCData_lib( DHCD )
+%DCData_lib( MAR )
+%DCData_lib( RealProp )
+
+%File_info( data=PresCat.TOPA_SSL, printobs=5 )
+%File_info( data=PresCat.TOPA_realprop, printobs=5 ) /** some IDs don't have real_prop info**/
+%File_info( data=PresCat.TOPA_addresses, printobs=5 )
+%File_info( data=PresCat.TOPA_database, printobs=5 ) /** 1699 obs**/
+
+/** merge address_id ssl and real_prop info **/
+
+data Topa_address_ssl_realprop;
+  merge
+    PresCat.TOPA_SSL (keep=id address_id ssl)
+    PresCat.TOPA_realprop (keep=id SALEDATE offer_sale_date CASD_date ADDRESS1 ADDRESS3 Ownername_full days_notice_to_sale ui_proptype Anc2012 cluster2017 Geo2020 GeoBg2020 GeoBlk2020 Psa2012 VoterPre2012 Ward2022 Ward2012)
+    PresCat.TOPA_addresses (keep =id address);
+run;
```

Comment:
@eburton9 You should have a BY statement whenever you use MERGE. Otherwise, the observations just match up based on their order in the respective data sets, which might not be correct. Did you want to match by ID?